apps/prairielearn/elements/pl-sketch/pl-sketch.py

```diff
@@ -0,0 +1,36 @@
+import chevron
+import lxml.html
+import prairielearn as pl
+
+
+def prepare(element_html, data):
```

Comment:
I think this function can just be deleted. Also no need to return data (is modified in-place).

Reply:
Wait, actually you should check for having the desired attributes on this element in prepare.